support/src/indirect_intrusive_heap.h

```diff
@@ -68,10 +68,10 @@ namespace crimson {
     class Iterator {
       friend IndIntruHeap<I, T, heap_info, C, K>;
 
-      IndIntruHeap<I, T, heap_info, C, K>& heap;
+      IndIntruHeap<I, T, heap_info, C, K>* heap;
       HeapIndex                            index;
 
-      Iterator(IndIntruHeap<I, T, heap_info, C, K>& _heap, HeapIndex _index) :
+      Iterator(IndIntruHeap<I, T, heap_info, C, K>* _heap, HeapIndex _index) :
 	heap(_heap),
```

Comment:
Change to "&_heap" to go from reference to pointer.

Reply:
In case it's not clear `heap(_heap)` will become `heap(&_heap)`.

Reply:
OK